app/controllers/students_controller.rb

```diff
@@ -30,14 +30,14 @@ def set_user
   end
 
   def user_params
-    validate_input.except(:student_information)
+    validate_input.except(:student_information_attributes)
```

Comment:
why are you excluding `student_information_attributes` from user params? I think this would work with the renaming you did for student information params, we can review this

Reply:
Agree ^